### ISSUE TITLE
It adds ellipses to even short articles.

### ISSUE THREAD

**Issue by crocket** [aprea/wp-advanced-excerpt#35]
Can it be fixed to add ellipses only to articles whose lengths exceed threshold?

**Comment by TheJester12** [comment]
Agreed, this is a problem. I use this plugin often when developing themes, but this is kind of a show-stopper. Right now I am using this code:

`the_advanced_excerpt('length=200&length_type=characters&finish=word&allowed_tags=strong');`
